.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "files.exclude": {
+        "**/*.js": { "when": "$(basename).ts" },
+        "**/*.map": { "when": "$(basename).map" },
+        "npm-debug.log": true,
+        "node_modules": true,
+        ".coverage": true,
+        ".test-results": true,
+        ".nyc_output": true,
+        ".taskkey": true,
+        ".release": true
```

Comment:
We usually hide the output/dist/etc. directories, but can remove this if you'd prefer to have this always available

Reply:
Totally agree. Again, it is what I have been doing locally.